cfme/control/explorer/conditions.py

```diff
@@ -181,7 +181,8 @@ def update(self, updates):
         view.fill(updates)
         view.save_button.click()
         view = self.create_view(ConditionDetailsView, override=updates)
-        assert view.is_displayed
+        wait_for(lambda: view.is_displayed, timeout=10,
+            message="wait until ConditionDetailsView will be displayed")
```

Comment:
**optional** consistent indent as per pep8 is either align with the `(` or put everything on a new lines